tapir/core/static/core/css/base.css

```diff
@@ -99,6 +99,12 @@ The css is from https://google.github.io/material-design-icons/#icon-font-for-th
   font-size: 16px;
 }
 
+.btn-outline-info {
```

Comment:
If `btn-outline-info` doesn't have enough contrast, I would prefer to use another button class rather than updating this one, so that we're sure that all the colors are correct. For example now, the color on hover doesn't fit.

Do `btn-primary` and `btn-primary-outline` work? It's just a different blue.
I find that `btn-dark` doesn't look very good, but maybe it's still the best option?
We could also simplify the color scheme, currently the distinction when to use `btn-outline-info` or `btn-outline-secondary` is not clear.
Maybe we just keep `btn-primary` for actual actions (like save), and use `btn-outline-dark` for all other buttons?

https://getbootstrap.com/docs/5.1/components/buttons/

Reply:
I followed the WCAG rule for contrasts and searched for a compatible color using a color palette. 
I agree with simplifying the color scheme now and create a new scheme as soon as we've figured it out.